sourcemash/frontend/assets.py

```diff
@@ -39,8 +39,10 @@
                 "js/views/feed_view.js",
                 "js/views/saved_view.js",
                 "js/views/browse_view.js",
+                "js/views/register_view.js",
```

Comment:
I just think i'd be annoyed if the modal pops up and the only action i can do is a "Godammit. Cancel."